diff -Nuar mc-4.7.5.3.orig/lib/tty/tty-ncurses.c mc-4.7.5.3/lib/tty/tty-ncurses.c
--- mc-4.7.5.3.orig/lib/tty/tty-ncurses.c	2015-04-15 16:43:18.000000000 +0300
+++ mc-4.7.5.3/lib/tty/tty-ncurses.c	2015-04-15 16:44:23.000000000 +0300
@@ -62,7 +62,7 @@
 /*** global variables ****************************************************************************/
 
 /*** file scope macro definitions ****************************************************************/
-
+#define _AIX
 #if defined(_AIX) && !defined(CTRL)
 #define CTRL(x) ((x) & 0x1f)
 #endif
