diff -ruN oscam-2015-11-26-11178.orig/Makefile oscam-2015-11-26-11178/Makefile
--- oscam-2015-11-26-11178.orig/Makefile	2015-10-01 00:11:35.000000000 +0300
+++ oscam-2015-11-26-11178/Makefile	2015-11-26 23:03:29.338911796 +0300
@@ -5,7 +5,7 @@
 .PHONY: all tests help README.build README.config simple default debug config menuconfig allyesconfig allnoconfig defconfig clean distclean
 
 VER     := $(shell ./config.sh --oscam-version)
-SVN_REV := $(shell ./config.sh --oscam-revision)
+#SVN_REV := $(shell ./config.sh --oscam-revision)
 
 uname_S := $(shell sh -c 'uname -s 2>/dev/null || echo not')
 
@@ -35,7 +35,7 @@
 	override USE_LIBCRYPTO=1
 endif
 
-CONF_DIR = /usr/local/etc
+CONF_DIR = /opt/etc/oscam
 
 LIB_PTHREAD = -lpthread
 LIB_DL = -ldl
