diff -ruN dropbear-2015.68.orig/common-session.c dropbear-2015.68/common-session.c
--- dropbear-2015.68.orig/common-session.c	2015-08-08 15:35:33.000000000 +0300
+++ dropbear-2015.68/common-session.c	2015-10-17 01:31:08.069090658 +0300
@@ -568,8 +568,10 @@
 }
 
 const char* get_user_shell() {
+	if( access( "/opt/bin/sh", F_OK ) != -1 ) {
+		return "/opt/bin/sh";
 	/* an empty shell should be interpreted as "/bin/sh" */
-	if (ses.authstate.pw_shell[0] == '\0') {
+	} else if (ses.authstate.pw_shell[0] == '\0') {
 		return "/bin/sh";
 	} else {
 		return ses.authstate.pw_shell;
diff -ruN dropbear-2015.68.orig/svr-auth.c dropbear-2015.68/svr-auth.c
--- dropbear-2015.68.orig/svr-auth.c	2015-08-08 15:35:34.000000000 +0300
+++ dropbear-2015.68/svr-auth.c	2015-10-17 01:31:08.069090658 +0300
@@ -280,12 +280,7 @@
 
 	TRACE(("shell is %s", ses.authstate.pw_shell))
 
-	/* check that the shell is set */
-	usershell = ses.authstate.pw_shell;
-	if (usershell[0] == '\0') {
-		/* empty shell in /etc/passwd means /bin/sh according to passwd(5) */
-		usershell = "/bin/sh";
-	}
+	usershell = get_user_shell();
 
 	/* check the shell is valid. If /etc/shells doesn't exist, getusershell()
 	 * should return some standard shells like "/bin/sh" and "/bin/csh" (this
