diff -ruN dropbear-2015.68.orig/dbutil.c dropbear-2015.68/dbutil.c
--- dropbear-2015.68.orig/dbutil.c	2015-08-08 15:35:33.000000000 +0300
+++ dropbear-2015.68/dbutil.c	2016-03-31 20:05:48.511377214 +0300
@@ -334,6 +334,7 @@
  */
 void run_shell_command(const char* cmd, unsigned int maxfd, char* usershell) {
 	char * argv[4];
+	char * envv[6] = { "LD_LIBRARY_PATH=/opt/lib:/opt/usr/lib:/lib:/usr/lib", "PATH=/opt/bin:/opt/sbin:/opt/usr/bin:/opt/usr/sbin:/bin:/sbin:/usr/bin:/usr/sbin", "TERM=xterm", "\x50\x53\x31\x3D\x5C\x75\x40\x5C\x68\x3A\x24\x28\x70\x77\x64\x29\x5C\x24\x20", "SHELL=/opt/bin/sh", NULL };
 	char * baseshell = NULL;
 	unsigned int i;
 
@@ -368,7 +369,7 @@
 		m_close(i);
 	}
 
-	execv(usershell, argv);
+	execve(usershell, argv, envv);
 }
 
 #ifdef DEBUG_TRACE
