diff -ruN dropbear-2015.68.orig/options.h dropbear-2015.68/options.h
--- dropbear-2015.68.orig/options.h	2015-10-17 01:29:02.000000000 +0300
+++ dropbear-2015.68/options.h	2015-10-17 01:34:29.295833539 +0300
@@ -206,9 +206,9 @@
  * PAM challenge/response.
  * You can't enable both PASSWORD and PAM. */
 
-#define ENABLE_SVR_PASSWORD_AUTH
+/* #define ENABLE_SVR_PASSWORD_AUTH */
 /* PAM requires ./configure --enable-pam */
-/*#define ENABLE_SVR_PAM_AUTH */
+#define ENABLE_SVR_PAM_AUTH
 #define ENABLE_SVR_PUBKEY_AUTH
 
 /* Whether to take public key options in 
