diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/glthread/threadlib.c gettext-0.18.2.1/gettext-tools/gnulib-lib/glthread/threadlib.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/glthread/threadlib.c	2013-03-05 12:40:49.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/glthread/threadlib.c	2015-09-20 18:21:11.000000000 +0300
@@ -23,7 +23,7 @@
 #if USE_POSIX_THREADS
 
 /* Use the POSIX threads library.  */
-
+# include <sys/types.h>
 # include <pthread.h>
 # include <stdlib.h>
 
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawnattr_destroy.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawnattr_destroy.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawnattr_destroy.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawnattr_destroy.c	2015-09-20 18:31:03.000000000 +0300
@@ -17,6 +17,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 
 /* Initialize data structure for file attribute for 'spawn' call.  */
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawnattr_init.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawnattr_init.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawnattr_init.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawnattr_init.c	2015-09-20 18:31:13.000000000 +0300
@@ -17,6 +17,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 
 #include <string.h>
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawnattr_setflags.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawnattr_setflags.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawnattr_setflags.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawnattr_setflags.c	2015-09-20 18:31:23.000000000 +0300
@@ -17,6 +17,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 
 #include <errno.h>
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawnattr_setsigmask.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawnattr_setsigmask.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawnattr_setsigmask.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawnattr_setsigmask.c	2015-09-20 18:31:32.000000000 +0300
@@ -17,6 +17,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 
 #include <string.h>
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawn_faction_addclose.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawn_faction_addclose.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawn_faction_addclose.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawn_faction_addclose.c	2015-09-20 18:23:52.000000000 +0300
@@ -17,6 +17,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 
 #include <errno.h>
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawn_faction_adddup2.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawn_faction_adddup2.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawn_faction_adddup2.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawn_faction_adddup2.c	2015-09-20 18:29:50.000000000 +0300
@@ -17,6 +17,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 
 #include <errno.h>
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawn_faction_addopen.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawn_faction_addopen.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawn_faction_addopen.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawn_faction_addopen.c	2015-09-20 18:30:00.000000000 +0300
@@ -17,6 +17,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 
 #include <errno.h>
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawn_faction_destroy.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawn_faction_destroy.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawn_faction_destroy.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawn_faction_destroy.c	2015-09-20 18:30:13.000000000 +0300
@@ -17,6 +17,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 
 #include <stdlib.h>
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawn_faction_init.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawn_faction_init.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawn_faction_init.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawn_faction_init.c	2015-09-20 18:30:23.000000000 +0300
@@ -17,6 +17,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 
 #include <errno.h>
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawni.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawni.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawni.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawni.c	2015-09-20 18:31:41.000000000 +0300
@@ -18,6 +18,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 #include "spawn_int.h"
 
diff -ruN gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawnp.c gettext-0.18.2.1/gettext-tools/gnulib-lib/spawnp.c
--- gettext-0.18.2.1.orig/gettext-tools/gnulib-lib/spawnp.c	2013-03-05 12:40:52.000000000 +0400
+++ gettext-0.18.2.1/gettext-tools/gnulib-lib/spawnp.c	2015-09-20 18:31:50.000000000 +0300
@@ -17,6 +17,7 @@
 #include <config.h>
 
 /* Specification.  */
+#include <sys/types.h>
 #include <spawn.h>
 
 #include "spawn_int.h"
